src/main/java/io/goobox/sync/storj/App.java

```diff
@@ -151,6 +154,11 @@ private void init(boolean resetAuthFile) {
         storj.setConfigDirectory(Utils.getDataDir().toFile());
         storj.setDownloadDirectory(syncDir.toFile());
 
+        if (!storj.keysExist()) {
+            logger.error("keyfile not found at the expected dir " + Utils.getDataDir().toFile());
+            System.exit(1);
```

Comment:
If no keys are present (the usual case after installation), the sync app will wait for a `login` command from the GUI. Exiting here will break this workflow.

Reply:
ohh okay. the gui is another code base? cause when i launch this app in eclipse, i don't see any ui?

Reply:
https://github.com/GooBox/goobox-community-gui